terms.html

```diff
@@ -93,10 +100,21 @@
   </dd>
   <dt><dfn data-lt="issuers|issuer's">issuer</dfn></dt>
   <dd>
-A role an <a>entity</a> can perform by asserting <a>claims</a> about one or
-more <a>subjects</a>, creating a <a>verifiable credential</a> from these
-<a>claims</a>, and transmitting the <a>verifiable credential</a> to a
-<a>holder</a>.
+A role an <a>party</a> can perform by <a>authoring</a> <a>claims</a>,
+constructing a <a>verifiable credential</a> from them,
+and transmitting that to a <a>holder</a>.
+  </dd>
+  <dt><dfn data-lt="party|parties|party's">party</dfn></dt>
+  <dd>
+An <a>entity</a> that has 'a mind of its own', i.e.,
+it sets the objectives it aims to pursue,
+it autonomously gathers, stores, processes and disseminates data (including <a>identifiers</a>),
+it subjectively decides on the semantics of that data (i.e., what the data means),
+it authors <a>claims</a>, manages its own logic, makes its own decisions, e.g., regarding
+the kinds of <a>credentials</a> to <a>issue</a>, 
+what data in such <a>credentials</a> it considers to be <a>valid</a> for paricular purposes,
+etc.
+Typical examples include people and organizations.
```

Comment:
Again, highlighting the reuse this technology enables is important. I'm not sure what to suggest here, but while it's good to say that an entity makes its own decisions, one of those decisions can be to reuse shared vocabularies and contexts that other entities have produced to ease their burden, increase interoperability, and enable better scalability.

Reply:
can you see where I put it?